dev-db/mysqltuner/mysqltuner-1.7.15.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=MySQLTuner-perl
+
+DESCRIPTION="MySQLTuner is a high-performance MySQL tuning script"
+HOMEPAGE="http://www.mysqltuner.com/"
+SRC_URI="https://github.com/major/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
It seems to be GPL-3+ ;) if you look at perl scripts in their Github.

Reply:
Done.